### DISCUSSION TITLE
Material em português - O que precisamos?

### DISCUSSION THREAD

**Discussion by edipofederle** [clj-br/forum#29] [Geral]
Olá pessoal, tudo bem?

Qual tipo de material seria interessante termos  disponivel em português para a comunidade brasileira de Clojure ? Estou pensando em começar a escrever algo, talvez já tenhamos alguns contúdo básicos por aí, mas seria interessente termos algo mais estruturado sobre o que pode ser útil para a comunidade, não apenas coisas básicas como introdução a linguagem, por example.

O que acham?

[]s

## Configuração e uso de IDEs/editores

- [ ] Emacs
- https://edipofederle.github.io/posts/2016-05-21-comecando-com-clojure-e-emacs.html
- [ ] vim
- [ ] VSCode
- https://imasters.com.br/desenvolvimento/utilizando-o-vs-code-para-programar-em-clojure
- [ ] IntelliJ
- [ ] Clojure no Windows
- https://dev.to/afucher/clojure-instalando-no-windows-496e

### Clojure beginner resources  [português]
[Parte 1: Conhecendo os building blocks da linguagem](https://medium.com/@alefeans/clojure-na-pr%C3%A1tica-parte-1-504c550a0d16);
[Parte 2: Polimorfismo, Multi-methods, Protocols e Records](https://alefeans.medium.com/clojure-na-pr%C3%A1tica-parte-2-3485c9ad7fea);
[Parte 3: Concorrência, paralelismo e gerenciamento de estado.](https://alefeans.medium.com/clojure-na-pr%C3%A1tica-parte-3-56115cf3ef0)

### Clojure beginner resources  [inglês]
- https://gist.github.com/yogthos/be323be0361c589570a6da4ccc85f58f

**Comment by ericdallo** [comment]
Boa idéia, sinto que material sobre IDEs/editors (Emacs, vim, VSCode, IntelliJ) seriam bem utéis, explicando como setupar o editor e codar em Clojure

**Comment by avelino** [comment]
emacs + clojure (`cider`): https://edipofederle.github.io/posts/2016-05-21-comecando-com-clojure-e-emacs.html

**Comment by avelino** [comment]
vscode + clojure por @woliveiras: https://imasters.com.br/desenvolvimento/utilizando-o-vs-code-para-programar-em-clojure

**Comment by geraldodev** [comment]
Como rodar o clojure no windows ? Eu imagino que esta seja uma barreira a
adoção. Como vocês utilizam no windows ? com WSL ?

Em qui., 17 de fev. de 2022 às 14:21, Avelino ***@***.***>
escreveu:

**Reply by avelino** [reply]
Clojure funciona super bem no Windows
https://github.com/clojure/tools.deps.alpha/wiki/clj-on-Windows

Lembrando que Clojure roda em cima da JVM e existe um trabalho constante do pessoal da OpenJVM para suportar Windows.

**Reply by geraldodev** [reply]
@avelino Você utiliza clojure no windows ? O doc que você referenciou fala de alpha-state, e historicamente o pessoal da cognitect não dedica recursos pra isso não. https://github.com/pedestal/pedestal#supported-platforms . 

De qualquer forma o objetivo era dizer que eu acho importante material de clojure relacionado ao windows apesar que na última vez que tentamos utilizar (faz tempo já) não obtivemos muito sucesso.

**Reply by afucher** [reply]
revivendo essa thread, to fazendo uns textos sobre Clojure e os dois primeiros foram fazendo setup no Windows: https://dev.to/afucher/series/21480
Vamos ver até quando vai ser tranquilo usar direto no Windows, sem WSL.

**Comment by alefeans** [comment]
@edipofederle eu publiquei uma série de artigos sobre Clojure na prática em português no Medium. Vou deixar os links aqui também:

[Parte 1: Conhecendo os building blocks da linguagem](https://medium.com/@alefeans/clojure-na-pr%C3%A1tica-parte-1-504c550a0d16);
[Parte 2: Polimorfismo, Multi-methods, Protocols e Records](https://alefeans.medium.com/clojure-na-pr%C3%A1tica-parte-2-3485c9ad7fea);
[Parte 3: Concorrência, paralelismo e gerenciamento de estado.](https://alefeans.medium.com/clojure-na-pr%C3%A1tica-parte-3-56115cf3ef0)
